api.lua

```diff
@@ -228,10 +247,7 @@ app:match('projects', '/projects', respond_to({
 
             if self.params.withthumbnail == 'true' then
                 for k, project in pairs(projects) do
-                    -- Lazy Thumbnail generation
-                    project.thumbnail =
-                        retrieve_from_disk(project.id, 'thumbnail') or
-                            generate_thumbnail(project.id)
+                    project.thumbnail = retrieveFromDisk(project.id, 'thumbnail')
```

Comment:
We can't take lazy thumbnail generation away. This will result in all migrated projects that haven't had their thumbnail generated to never load it. The alternative is to run a 20-something hour long script that generates all thumbnails, which makes no sense given they can be generated on the fly for almost no cost.